### ISSUE TITLE
Mercury for Flu enhancement - label lineage pdm09 for H1N1 samples

### ISSUE THREAD

**Issue by kapsakcj** [theiagen/public_health_bioinformatics#589]
:cool:

### :pushpin: Explain the Request

We recently added functionality (via PR #506 and PR3 over in Mercury repo: https://github.com/theiagen/mercury/pull/3) in Mercury to automatically generate the NCBI BioSample metadata field `isolate` in the following format:

```
# format:
type/state/submission_id/year (subtype)

# example:
A/California/sample-0001/2024 (H1N1)
```

A user requested that the string `pdm09` is appended to the string auto-generated in the `isolate` column, so for example `A/California/sample-0001/2024 (H1N1)pdm09`. This is to designate that samples are descendants/related to the pandemic 2009 strain.

However, we currently do not have a way in TheiaCoV wfs to determine if samples are truly descendants or not. 

Also, not all H1N1 strains are descendants (like those collected prior to 2009) or H1N1 samples collected in the future could potentially  not be part of the pdm09 lineage.

It would be beneficial to our users if this designation could be determined (perhaps via Nextclade or another mechanism/algorithm like something involving collection dates) and appended to the string auto-generated in Mercury code.

The current workaround is to run Mercury and manually edit the BioSample metadata TSV output from the workflow to include this lineage designation prior to NCBI submission

**Comment by kapsakcj** [comment]
This wasn't completed, so I'm reopening it

**Comment by sage-wright** [comment]
sorry! this was a misclick

**Comment by AndrewLangvt** [comment]
@kapsakcj given there is not a readily-available mechanism to assess & assign pdm09 lineage relation, I'm inclined to close this issue. Have you had additional conversations with the requesting lab/other labs raise a similar request?

**Comment by kapsakcj** [comment]
No, no discussions on this since it was originally brought up, we are OK to close. Thanks for checking 👍 

The workaround is to just manually edit the mercury output TSVs to add the pdm09 designation where appropriate.
